src/components/common/YourCompany/ResolutionDate.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <section id="resolution-date">
+    <v-row no-gutters>
+      <v-col cols="3" class="pr-2">
+        <label :class="{'error-text': invalidSection}">
+          <strong>Resolution Date</strong>
+        </label>
+        <v-chip
+          v-if="isCorrected"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          <span>{{ editedLabel }}</span>
+        </v-chip>
+      </v-col>
+
+      <!-- Info tooltip-->
+      <v-col cols="9">
+        <span v-if="!onEditMode" class="info-text mr-1">{{ resolutionDate }}</span>
+        <v-tooltip
+          top
+          content-class="top-tooltip"
+          transition="fade-transition"
+          nudge-right="3"
+        >
+          <template v-slot:activator="{ on }">
+            <v-icon v-on="on" class="info-icon">mdi-information-outline</v-icon>
+          </template>
+          <span>If the resolution date is incorrect, it must be corrected through a correction filing.</span>
```

Comment:
Haha, that's exactly what we're trying to do with this ticket: allow corrections in this correction filing.